network-api/networkapi/templates/pages/buyersguide/article_page.html

```diff
@@ -47,19 +47,22 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
     </div>
   </div>
 
+  {% for block in page.body %}
+      {% include_block block with parent_page=page page_type="blog" %}
+  {% endfor %}
+
   {% with related_articles=page.get_primary_related_articles  %}
     {% if related_articles %}
-      {% trans "PRIMARY RELATED ARTICLES" %}
-      <div class="tw-m-4">
-        {% include "fragments/buyersguide/related_reading.html" with articles=related_articles %}
+    <div class="tw-container tw-my-4">
+      <div class="tw-row">
+        <div class="tw-px-4 tw-w-8/12 tw-mx-auto" id="pni-related-content">
```

Comment:
What do you think about using `related-articles` as the id? That would feel more consistent with the variables and template names.

Reply:
I am also wondering about the `pni-` prefix. Can you add some context why we need this?

Reply:
I do not think we should use `related-articles` over `pni-related-content` since it is referring to the parent div of the related article template.

I have done the pni- prefix for other templates on PNI just incase I think the name is too generic. I did this for some of the new dropdowns as well. I think we have had some issues in the past where over generic names for classes/ids have caused bugs and unintended changes so I think it is better to be more specific then not.

Reply:
Thanks for expanding on this. 

What do you think about using `buyersguide` as the prefix if you think this needs to be more specific? I find the mixed use of `pni` and `buyersguide` rather confusing. (E.g. all the templates are in a directory called `buyersguide` but the id uses `pni`.)

Reply:
Does that mean we want to change all `pni-` prefixes to `buyersguide-`? That is a quite a bit of classes names to change including tailwind color palettes? I would almost think changing the buyersguide references would be easier but either or would be more work then the original ticket. We could open a ticket about this and discuss why we should do one over the other but I do not think this large of a refactor should be included on this PR.

Reply:
Yea, I think it would make sense to change all the `pni-` occurrences to `buyersguide-` since all files as well as the page types and other classes use that label. Of course this does not need to happen all in this PR. But I think when we are already touching something (or adding something new like in this case) we should strive toward more consistency.

Reply:
Okay, then I will update the ones currently added on this PR.